wallet/res/layout/dialog_new_account_confirm.xml

```diff
@@ -31,25 +31,86 @@
+            android:layout_gravity="center_horizontal"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/input_code_dash"
+                android:layout_width="28dp"
+                android:layout_height="28dp"
+                android:layout_gravity="center_vertical"
+                android:layout_marginTop="2dp"
+                android:layout_marginEnd="8dp"
+                app:srcCompat="@drawable/ic_dash_d_black" />
+
+            <TextView
+                android:id="@+id/input_value"
+                style="@style/MontserratRegular"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:text="0.00"
+                android:textColor="@color/dash_black"
+                android:textSize="36sp" />
+
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/fiat_symbol"
+                style="@style/MontserratRegular"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="$"
+                android:textColor="@color/dash_gray"
+                android:textSize="24sp" />
+
+            <TextView
+                android:id="@+id/fiat_value"
+                style="@style/MontserratRegular"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="0.00"
+                android:textColor="@color/dash_gray"
+                android:textSize="24sp" />
+
+        </LinearLayout>
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/message"
+        style="@style/TextViewStyle.Body"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:padding="16dp">
-
-        <TextView
-            android:id="@+id/message"
-            style="@style/TextViewStyle.Body"
-            android:layout_width="match_parent"
-            android:layout_height="0dp"
-            android:gravity="center"
-            android:text="It costs D0.01 to create your Evolution account."
-            android:textColor="@color/dash_black"
-            app:layout_constraintDimensionRatio="3:1"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
-
-    </androidx.constraintlayout.widget.ConstraintLayout>
+        android:gravity="center"
+        android:padding="16dp"
+        android:text="You have chose “Johndoe” as your username. Username cannot be changed once its registered."
```

Comment:
Should we move this to a resource string and format it with the chosen username?

Reply:
Oh, it is only for preview, I should probably use `tools:text` property instead of `android:text`.
I'll change this. The value of this field is being set with the real data over here: https://github.com/dashevo/dash-wallet/pull/369/files/eb371b8504c380c3e508fd210471ab1eef17a350